libvips/foreign/dzsave.c

```diff
@@ -116,9 +116,9 @@
 
 	Overlap handling
 
-   For deepzoom, tile-size == 256 and overlap == 1 means that edge tiles are 
-   257 x 257 (though less at the bottom right) and non-edge tiles are 258 x 
-   258. Tiles are positioned across the image in tile-size steps. This means 
+   For deepzoom, tile-size == 254 and overlap == 1 means that edge tiles are 
+   255 x 255 (though less at the bottom right) and non-edge tiles are 256 x 
+   256. Tiles are positioned across the image in tile-size steps. This means 
    (confusingly) that two adjoining tiles will have two pixels in common.
```

Comment:
With the explanation of the purpose of tile overlap, hopefully the `(confusingly)` is not necessary anymore. If not, let me know and I’d be happy to expand based on my knowledge :eyes: